proposals/description/tip_121c.ts

```diff
@@ -67,87 +103,16 @@ const tip_121c: TemplatedProposalDescription = {
       description: 'Burn FEI held on the new DAI PSM'
     },
 
-    // 2a. Revoke deprecated MINTER_ROLE roles
+    // 2. Revoke deprecated MINTER_ROLE role from timelock
     {
       target: 'core',
       values: '0',
       method: 'revokeMinter(address)',
       arguments: (addresses) => [addresses.feiDAOTimelock],
       description: 'Revoke MINTER_ROLE from the DAO Timelock.'
     },
-    {
-      target: 'core',
-      values: '0',
-      method: 'revokeMinter(address)',
```

Comment:
pcvEquityMinter deprecation is done in on-going cleanup DAO vote

Reply:
ok